proposals/NNNN-Read-Generic-Network-Signal-data.md

```diff
@@ -0,0 +1,292 @@
+# Read Generic Network Signal data
+
+* Proposal: [SDL-NNNN](NNNN-Read-Generic-Network-Signal-data.md)
+* Author: [Ankur Tiwari](https://github.com/ATIWARI9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
+
+## Introduction
+
+To enable OEM proprietary mobile apps to read raw CAN/Network data by specifying uniquely identifiable network data attributes. This expands the utilization of SDL APIs to read all the network signals available to head unit.
+
+## Motivation
+
+* Vehicles are getting more capable now than ever with advanced sensors and connectivity options. This generates tremendous amount of usable data for mobile apps in connectivity and mobility solutions domain. In order to keep up with varying demand and changing trends, SDL needs to be able to upscale   with access to new vehicle data parameters without having to wait for lengthy software build cycle for head units.
```

Comment:
Please remove the extra space between "upscale" and "with" in the last sentence, and update the end of the last sentence to "...wait for a lengthy software build cycle..." (adding "a").